exercises/practice/connect/.docs/instructions.md

```diff
@@ -0,0 +1,31 @@
+# Instructions
+
+Compute the result for a game of Hex / Polygon.
+
+The abstract boardgame known as
```

Comment:
```suggestion
The abstract board game known as
```

Reply:
If this is from upstream (problem-specifications repository) then no problems, it should be "corrected" there.